src/Tokenizer/Analyzer/Analysis/ArgumentAnalysis.php

```diff
@@ -20,25 +20,21 @@
 final class ArgumentAnalysis
 {
     /**
-     * The default value of the argument.
-     *
-     * @var null|string
+     * The name of the argument.
      */
-    private $default;
+    private string $name;
 
     /**
-     * The name of the argument.
-     *
-     * @var string
+     * The index where the name is located in the supplied Tokens object.
      */
-    private $name;
+    private int $nameIndex;
 
     /**
-     * The index where the name is located in the supplied Tokens object.
+     * The default value of the argument.
      *
-     * @var int
+     * @var null|string
      */
-    private $nameIndex;
+    private $default;
```

Comment:
i'm confused about changed order of properties

like, is it that we have some ordering of typed vs non-typed properties automated, @SpacePossum ?

Reply:
Likely custom fixer that orders the properties based on the arguments/assignment in the constructor.